torchx/c_src/torchx.cpp

```diff
@@ -313,19 +313,6 @@ NIF(concatenate)
 }
 
 NIF(gather)
-{
```

Comment:
I've removed `take_along_axis` to expose the actual torch gather semantics instead